pkg/crc/machine/bundle/extract.go

```diff
@@ -1,29 +1,36 @@
 package bundle
 
 import (
+	"archive/tar"
+	"github.com/xi2/xz"
 	"io"
 	"os"
 	"path/filepath"
-
-	"archive/tar"
-	"compress/gzip"
+	"strings"
 )
 
-func Extract(sourcepath string, destpath string) error {
+func Extract(sourcepath string, destpath string) (string, error) {
 	file, err := os.Open(sourcepath)
 
 	if err != nil {
-		return err
+		return "", err
+	}
+
+	extractedPath := strings.Split(filepath.Base(file.Name()), ".tar")[0]
+	extractedPath = filepath.Join(destpath, extractedPath)
```

Comment:
so what if someone changes the name of the bundle?

Reply:
This info should come from the file type that gets extracted. it contains the actual path and this needs to be split... and not the archive file name.